servicetalk-transport-netty-internal/src/main/java/io/servicetalk/transport/netty/internal/NettyLoggerUtils.java

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright © 2020 Apple Inc. and the ServiceTalk project authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.servicetalk.transport.netty.internal;
+
+import io.netty.handler.logging.LogLevel;
+import io.netty.util.internal.logging.InternalLogger;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import static io.netty.handler.logging.LogLevel.DEBUG;
+import static io.netty.handler.logging.LogLevel.ERROR;
+import static io.netty.handler.logging.LogLevel.INFO;
+import static io.netty.handler.logging.LogLevel.TRACE;
+import static io.netty.handler.logging.LogLevel.WARN;
+
+/**
+ * Utility methods for {@link InternalLogger} related types.
+ */
+public final class NettyLoggerUtils {
+    private NettyLoggerUtils() {
+    }
+
+    /**
+     * Translate the log level as determined by SL4J for the {@code loggerName} to Netty's {@link LogLevel} type.
+     * @param loggerName The logger name to lookup.
+     * @return Netty's {@link LogLevel} corresponding to the SL4J log level for {@code loggerName}.
+     */
+    public static LogLevel getNettyLogLevel(String loggerName) {
```

Comment:
A behavior that is useful is to enable wire logging in code but disable it using logger levels. I *think* this will always enable wire logging if it is enabled in code, is that correct?

Reply:
Good question! Most of the logger implementations also support `OFF` that disables logging [1, 2] from the logger config. But we need to adjust `getNettyLogLevel` method to support it.

1. https://logging.apache.org/log4j/2.x/log4j-api/apidocs/org/apache/logging/log4j/Level.html#OFF
2. http://logback.qos.ch/apidocs/ch/qos/logback/classic/Level.html#OFF

Reply:
The configuration can be modified to accommodate this, an appender can include a threshold level. I will enhance our test `log4j2-test.xml` to accommodate this.

Reply:
I am not sure I am following the motivation now. Threshold filter is specific to log4j, so not ideal that we have to depend on that for the expected behavior.

Why not just let the user specify the level at which they want to log?

Reply:
Threshold filter was one example of how this can be handled in configuration, setting the logging value to OFF is another example. Specifying the log level programmatically requires additional API surface area (e.g. log level API, propagation of log level) which seems redundant if the configuration has to be setup to enable the logger.